### DISCUSSION TITLE
modprobe: FATAL: Module snd-dummy not found in directory /lib/modules/6.2.0-1011-azure

### DISCUSSION THREAD

**Discussion by pheiduck** [actions/virtual-environments#9074] [General questions]
### Description

snd-dummy is not available on Linux 6.2

### Platforms affected

- [x] Azure DevOps
- [X] GitHub Actions - Standard Runners
- [X] GitHub Actions - Larger Runners

### Runner images affected

- [ ] Ubuntu 20.04
- [X] Ubuntu 22.04
- [ ] macOS 11
- [ ] macOS 12
- [ ] macOS 13
- [ ] Windows Server 2019
- [ ] Windows Server 2022

### Image version and build link

Version: 20230911.1.0

### Is it regression?

yes: https://github.com/vim/vim/actions/runs/6162407490/job/16723803467

### Expected behavior

no error

### Actual behavior

https://github.com/vim/vim/actions/runs/6163868619/job/16728348781#step:10:2611

### Repro steps

Run CI job which have `linux-modules-extra-6.2.0-1011-azure` installed `modprobe snd-dummy`

**Comment by shamil-mubarakshin** [comment]
Hello @pheiduck,
Thanks for reporting. Images are built upon Azure Marketplace offerings, which recently had their ubuntu22's kernel updated to `6.2.0-1011-azure`. We will reach out to canonical and see if this can be fixed in next releases.

**Comment by pheiduck** [comment]
@shamil-mubarakshin any news?

**Comment by shamil-mubarakshin** [comment]
@pheiduck, unfortunately there are no news yet

**Comment by isZumpo** [comment]
I'm also having this issue still. Any news on the topic?

**Reply by StephenHodgson** [reply]
also running into this

```
sudo modprobe snd-dummy
modprobe: FATAL: Module snd-dummy not found in directory /lib/modules/6.5.0-1025-azure
```
